### PR TITLE
Fix/add where we work locations

### PR DESCRIPTION
**Pull request by davidebukali**
Fix for [360](https://github.com/devinit/DIwebsite-redesign/issues/360)

### CODE REVIEW

File: di_website/whatwedo/blocks.py [hunk 1]
```diff
@@ -81,3 +82,24 @@ class ExpertiseBlock(StructBlock):
...
     class Meta():
         template = 'blocks/expertise.html'
+
+
+class MapBlock(StructBlock):
+    """
+    Allows the addition of a single benefit
+    """
+    location = TextBlock()
+    address = TextBlock()
```
Comment (edwinmp) [inline di_website/whatwedo/blocks.py]: Should this be required?

File: di_website/whatwedo/blocks.py [hunk 2]
```diff
@@ -81,3 +82,24 @@ class ExpertiseBlock(StructBlock):
 
     class Meta():
         template = 'blocks/expertise.html'
+
+
+class MapBlock(StructBlock):
```
Comment (edwinmp) [inline di_website/whatwedo/blocks.py]: Could you please rename this to LocationBlock

File: di_website/whatwedo/blocks.py [hunk 3]
```diff
@@ -81,3 +82,24 @@ class ExpertiseBlock(StructBlock):
...
+    """
+    Allows the addition of a single benefit
+    """
+    location = TextBlock()
```
Comment (edwinmp) [inline di_website/whatwedo/blocks.py]: `name`

File: di_website/whatwedo/blocks.py [hunk 4]
```diff
@@ -81,3 +82,24 @@ class ExpertiseBlock(StructBlock):
 
     class Meta():
         template = 'blocks/expertise.html'
+
+
+class MapBlock(StructBlock):
+    """
+    Allows the addition of a single benefit
```
Comment (edwinmp) [inline di_website/whatwedo/blocks.py]: `a single location`

File: di_website/whatwedo/blocks.py [hunk 5]
```diff
@@ -81,3 +82,24 @@ class ExpertiseBlock(StructBlock):
...
+
+    class Meta():
+        icon = 'fa-map'
+
+
+class MapStreamBlock(StreamBlock):
+    """
+    Handles creation of where we work locations
+    """
+    add_location = MapBlock()
```
Comment (edwinmp) [inline di_website/whatwedo/blocks.py]: Just `location` is fine I think... can add any verbose naming for the UI in the meta class

File: di_website/whatwedo/blocks.py (hunk 1 above)
Comment (davidebukali) [reply di_website/whatwedo/blocks.py]: `address = TextBlock()` should not be required, however `location = TextBlock()` is needed on the map to show the country name on mouse hover.

Comment (davidebukali) [reply di_website/whatwedo/blocks.py]: Just replaced `addresss` with an optional `mobile` field` which can be also be shown on the map.

File: di_website/whatwedo/blocks.py (hunk 2 above)
Comment (davidebukali) [reply di_website/whatwedo/blocks.py]: Just done and pushed

File: di_website/whatwedo/blocks.py (hunk 3 above)
Comment (davidebukali) [reply di_website/whatwedo/blocks.py]: Updated and pushed

File: di_website/whatwedo/blocks.py (hunk 4 above)
Comment (davidebukali) [reply di_website/whatwedo/blocks.py]: Just updated

File: di_website/whatwedo/blocks.py (hunk 5 above)
Comment (davidebukali) [reply di_website/whatwedo/blocks.py]: Also updated

**Pull request merged**